src/Language/PureScript/Docs/Types.hs

```diff
@@ -413,6 +413,7 @@ asDeclarationInfo = do
     "typeClass" ->
       TypeClassDeclaration <$> key "arguments" asTypeArguments
                            <*> key "superclasses" (eachInArray asConstraint)
+                           <*> key "fundeps" asFunDeps
```

Comment:
Good point. I think we could use `keyOrDefault "fundeps" [] asFunDeps` here, and then JSON produced by older compilers could still be submitted to Pursuit - any docs produced by older compilers would then be missing functional dependencies, but otherwise would still work.